py/runtime.c

```diff
@@ -73,6 +73,8 @@ void mp_init(void) {
 #endif
 
     #if MICROPY_KBD_EXCEPTION
+    // Disable intercepting any interrupt character
+    MP_STATE_VM(interrupt_char) = -1;
```

Comment:
Should this be `MP_INTERRUPT_CHAR_DISABLE_AND_CANCEL_PENDING` instead of `-1`?

Reply:
Yes indeed!

Reply:
Now fixed.